app/lib/ui/views/active/atr_editing_screen.dart

```diff
@@ -185,6 +185,46 @@ class _ATREditingScreenState extends State<ATREditingScreen> {
                         style: TextStyle(
                             color: NavyBlue, fontWeight: FontWeight.bold),
                       ),
+                      actions: [
+                        IconButton(
+                            icon: Icon(Icons.delete),
+                            color: NavyBlue,
+                            onPressed: () => showDialog(
```

Comment:
- [ ] I think we can use `DialogScreen` here with `dialogueService.showDialogue(confirm: true)` or similar...
- [ ] Move this whole thing into a separate function vs into one long scaffold (like `_saveBeforeWillPop` but this could be `_confirmAndDeleteAtr`)...